compat/test/browser/select.test.js

```diff
@@ -0,0 +1,32 @@
+import { setupScratch, teardown } from '../../../test/_util/helpers';
+import { render, createElement as h } from '../../src';
+
+/** @jsx h */
+
+describe('Select', () => {
+	let scratch;
+
+	beforeEach(() => {
+		scratch = setupScratch();
+	});
+
+	afterEach(() => {
+		teardown(scratch);
+	});
+
+
+	it('should work with multiple selected (array of values)', () => {
+		function App() {
+			return (
+				<select multiple value={['B', 'C']}>
+					<option value="A">A</option>
+					<option value="B">B</option>
+					<option value="C">C</option>
+				</select>
+			);
+		}
+
+		render(<App />, scratch);
+		expect(scratch.firstChild.value).to.equal('B');
```

Comment:
Let's check that both `options B` and `options A` is selected here to make sure it works :+1: